plutus-conformance/agda/Spec.hs

```diff
@@ -37,8 +38,57 @@ agdaEvalUplcProg (UPLC.Program () version tmU) =
 
 -- | These tests are currently failing so they are marked as expected to fail.
 -- Once a fix for a test is pushed, the test will fail. Remove it from this list.
+-- The entries of the list should be paths from the root of plutus-conformance
+-- to the directory containing the test, eg
+--   "test-cases/uplc/evaluation/builtin/semantics/addInteger/addInteger1"
+
 failingTests :: [FilePath]
-failingTests = [
+failingTests =
+    [
+     --- byteStringToInteger
+      "test-cases/uplc/evaluation/builtin/semantics/byteStringToInteger/big-endian/all-zeros"
```

Comment:
It says `failingTests`, but `all-zeros` isn't a failing test? I'm confused.

Reply:
> It says failingTests, but all-zeros isn't a failing test? I'm confused.

These do fail in the metatheory because in this branch the Agda code doesn't know about the new builtins yet.  That'll be fixed in a PR that'll appear shortly (and which I should have done before this one).  In that branch `failingTests` is empty.

Reply:
If you run the `agda-conformance` tests you get errors like
```
          big-endian
            all-zeros: FAIL (expected)
              Agda: unreachable code reached.
              CallStack (from HasCallStack):
                error, called at /blah/blah/blah/plutus-metatheory-0.1.0.0/build/MAlonzo/RTE.hs:44:23 in plutus-metatheory-0.1.0.0-inplace:MAlonzo.RTE (expected failure)

Reply:
I see, thank you for the explanation.